usmqe/web/tendrl/clusters/pages.py

```diff
@@ -69,3 +81,35 @@ class ClustersList(contentviews.ListView):
     _model = m_cluster_list.ClustersListModel
     _label = 'main page - clusters - list'
     _row_class = ClustersRow
+
+
+class ClustersHostsRow(contentviews.ListViewRow):
+    """
+    Cluster in Clusters list
+    """
+    _model = m_cluster_list.ClustersHostsRowModel
+    _required_elems = []
+
+    @property
+    def host(self):
+        """ returns cluster name """
+        return self._model.host.text
+
+
+class ClustersHostsList(contentviews.ListView):
```

Comment:
It will look better if you use existing `CloseDlg` class from webstr
https://github.com/Webstr-framework/webstr/blob/master/webstr/common/dialogs/pages.py